Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs

```diff
@@ -6,12 +6,12 @@
 using System.Runtime.InteropServices;
 using UnityEngine;
 
-namespace HoloToolkit.Unity
+namespace MixedRealityToolkit.Utilities.GpuTiming
 {
-	/// <summary>
-	/// Encapsulates access to GPU timing methods.
-	/// </summary>
-	public static class GpuTiming
+    /// <summary>
```

Comment:
Looks like these tabs were turned into spaces, but not the rest of the file (still tabs). Should we go ahead and convert the rest of this file (and GpuTimingCamera.cs)?

Reply:
I'm all in favour of not mixing tabs / spaces.  Odd it should have changed though.

Reply:
Ok, pushing a fix to resolve tabs for spaces across all the scripts. As that seemed to be the standard.
Might up the change file count.  But seemed to be limited to scripts that were brought in to the project externally.

Reply:
spaces > tabs